recipes/cudatoolkit-dev/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "cudatoolkit-dev" %}
+{% set version = "9.2" %}
+{% set version_build = "148" %}
+{% set driver_version = "396.37" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  path: ./
+
+build:
+  script: python build.py {{ version }} {{ version_build }} {{ driver_version }}
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  host:
+   - python >=3
+
+  run:
+   - python >=3
+   - 7za  # [win]
+   - conda >=4.5.8
```

Comment:
How does this work? Can it only be installed in `root` then?

Reply:
@jakirkham, 
1. The post install script uses two functions `download()`, and `hashsum_file()` from the conda package. This is why it is listed as a requirement. 
2. It can be installed in any conda environment.